README.md

```diff
@@ -33,7 +33,7 @@ Use the following table to verify you have the correct prerequisites to install
 		<td><a href="http://devdocs.magento.com/guides/v2.0/install-gde/prereq/apache.html">Apache</a></td>
 	</tr>
 	<tr>
-		<td>PHP ~5.5.22 or 5.6.x</td>
```

Comment:
Should be ~7.0.2 I believe:
- http://devdocs.magento.com/guides/v2.0/install-gde/prereq/php-ubuntu.html
- http://devdocs.magento.com/guides/v2.0/install-gde/prereq/php-centos.html